### ISSUE TITLE
Remmina 1.3.7 missing RDP plugin

### ISSUE THREAD

**Issue by sahsanu** [flathub/org.remmina.Remmina#11]
Hello,

A few minutes ago remmina's flatpak has been updated from version 1.3.6 to 1.3.7 and now I can't connect to my RDP sessions, I get the following message "_Install the RDP protocol plugin first_"

Running from shell I can see the following message:
```
Failed to load plugin: /app/lib/remmina/plugins/remmina-plugin-rdp.so.
Error: /app/lib/remmina/plugins/remmina-plugin-rdp.so: undefined symbol: freerdp_settings_set_string

```
Update: Added remmina package info.
```
$ flatpak info org.remmina.Remmina
Ref: app/org.remmina.Remmina/x86_64/stable
ID: org.remmina.Remmina
Arch: x86_64
Branch: stable
Origin: flathub
Collection ID: org.flathub.Stable
Date: 2019-12-08 21:01:00 +0000
Subject: Upstream release v1.3.7 (79005a84)
Commit: 92cceba4c131c3e2ccc21af75e6706958daba8ee6656b5ce49c217b1142b1104
Parent: c931de656f9d00336181b441567e945a0db01e6f06010e8dd5437cf1e7feeba7
Location: /var/lib/flatpak/app/org.remmina.Remmina/x86_64/stable/92cceba4c131c3e2ccc21af75e6706958daba8ee6656b5ce49c217b1142b1104
Installed size: 12,7 MB
Runtime: org.gnome.Platform/x86_64/3.34
Sdk: org.gnome.Sdk/x86_64/3.34
```

Appreciate your support.

**Comment by antenore** [comment]
@larchunix can we use the FreeRDP master? The latest tag is way too old

**Comment by SupercowsNL** [comment]
Same issue here.. rdp pluggin not working anymore

**Comment by antenore** [comment]
I've switched temporary to the master branch of FreeRDP.
We will see how to better address this issue ASAP.
The build is in progress, please test (in half an hour) and let me know 

cc: @SupercowsNL @larchunix

**Comment by sahsanu** [comment]
@antenore, I've updated it 5 minutes ago and RDP sessions are working fine again.

Package info:
```
$ flatpak info org.remmina.Remmina                                                                                             0 < 11:25:58
Ref: app/org.remmina.Remmina/x86_64/stable
ID: org.remmina.Remmina
Arch: x86_64
Branch: stable
Origin: flathub
Collection ID: org.flathub.Stable
Date: 2019-12-09 09:20:33 +0000
Subject: Using FreeRDP master to temporary fix for #11 (86714014)
Commit: 33ff2a37022a4c6a505f5e2a6c34fbba5bf5d18fd3460c52a324ca66076dea85
Parent: 92cceba4c131c3e2ccc21af75e6706958daba8ee6656b5ce49c217b1142b1104
Location: /var/lib/flatpak/app/org.remmina.Remmina/x86_64/stable/33ff2a37022a4c6a505f5e2a6c34fbba5bf5d18fd3460c52a324ca66076dea85
Installed size: 12,8 MB
Runtime: org.gnome.Platform/x86_64/3.34
Sdk: org.gnome.Sdk/x86_64/3.34
```

Thank you very much.

**Comment by SupercowsNL** [comment]
Same here! thx for fixing it so fast  :-)
